src/settings/Settings.ts

```diff
@@ -331,11 +331,11 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         displayName: _td('Mirror local video feed'),
         default: false,
     },
-    "TagPanel.enableTagPanel": {
+    "GroupFilterPanel.enableGroupFilterPanel": {
```

Comment:
this is a breaking change - we either need to be backwards compatible in the setting handlers or not rename the setting for now.